#!/usr/bin/env python

PACKAGE = "ptcam_arduino"

from dynamic_reconfigure.parameter_generator_catkin import *
	
gen = ParameterGenerator()

gen.add("roi_minarea", double_t, 0, "min area of ROI [%]", 0.04, 0.01, 1.0)
gen.add("pan_gain", int_t, 0, "pan gain", 25, 1, 100)
gen.add("pan_dzone", double_t, 0, "pan dead zone[deg]", 4.0, 0.5, 30.0)
gen.add("tilt_gain", int_t, 0, "tilt gain", 25, 1, 100)
gen.add("tilt_dzone", double_t, 0, "tilt dead zone[deg]", 4.0, 0.5, 30.0)

exit(gen.generate(PACKAGE, "follower", "Follower"))

